pom.xml

```diff
@@ -367,24 +364,29 @@
           org.jboss.hal.testsuite.category.PicketBox,
           org.jboss.hal.testsuite.category.PicketLink
         </test.included.category>
-        <test.excluded.category>
-          org.jboss.hal.testsuite.category.KnownIssue
-        </test.excluded.category>
       </properties>
     </profile>
 
     <profile>
       <id>noSecurity</id>
       <properties>
         <test.excluded.category>
-          org.jboss.hal.testsuite.category.KnownIssue,
           org.jboss.hal.testsuite.category.Elytron,
           org.jboss.hal.testsuite.category.PicketBox,
           org.jboss.hal.testsuite.category.PicketLink
         </test.excluded.category>
       </properties>
     </profile>
 
+    <profile>
+      <id>withoutKnownIssues</id>
+      <properties>
+        <test.excluded.category>
```

Comment:
There seems to be conflict of `test.excluded.category` property between `noSecurity` and `withoutKnownIssues` profiles. If you use them both it would be IMO overwritten.